src/lib/containers/CardContainer.tsx

```diff
@@ -98,7 +98,14 @@ export const CardContainer = (props: CardContainerProps) => {
     // data was retrieved from the backend but there is none to show, we return a empty fragment
     // there could be a more informed UX decision here but the current use case right now
     // should display nothing
```

Comment:
I think you can get rid of this above comment 🙂

Reply:
Nice catch, updated comment.